[2024-06-24 14:28:21,715] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2023-02-19T00:00:00+00:00 [queued]>
[2024-06-24 14:28:21,804] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2023-02-19T00:00:00+00:00 [queued]>
[2024-06-24 14:28:21,814] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 14:28:21,817] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-06-24 14:28:21,846] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 14:28:21,897] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2023-02-19T00:00:00+00:00
[2024-06-24 14:28:21,907] {standard_task_runner.py:52} INFO - Started process 15897 to run task
[2024-06-24 14:28:21,912] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2023-02-19T00:00:00+00:00', '--job-id', '7018', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp23mckwbr', '--error-file', '/tmp/tmpvb02rco6']
[2024-06-24 14:28:21,920] {standard_task_runner.py:77} INFO - Job 7018: Subtask second_task
[2024-06-24 14:28:22,053] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2023-02-19T00:00:00+00:00 [running]> on host 799b1e36e3f1
[2024-06-24 14:28:22,319] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2023-02-19T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-19T00:00:00+00:00
[2024-06-24 14:28:22,321] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-06-24 14:28:22,325] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2024-06-24 14:28:22,386] {bash.py:169} INFO - Output:
[2024-06-24 14:28:22,394] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2024-06-24 14:28:22,396] {bash.py:177} INFO - Command exited with return code 0
[2024-06-24 14:28:22,598] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20230219T000000, start_date=20240624T142821, end_date=20240624T142822
[2024-06-24 14:28:22,702] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-24 14:28:22,762] {local_task_job.py:146} INFO - Task exited with return code 0
